sdk/easm/azure-analytics-defendereasm/pom.xml

```diff
@@ -0,0 +1,125 @@
+<!--
+ ~ Copyright (c) Microsoft Corporation. All rights reserved.
+ ~ Licensed under the MIT License.
+ ~ Code generated by Microsoft (R) AutoRest Code Generator.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure</groupId>
```

Comment:
Parent pom reference is missing. Please add the parent pom as [shown here](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/appconfiguration/azure-data-appconfiguration/pom.xml#L7-L12).